packages/solid-query/package.json

```diff
@@ -54,7 +54,7 @@
     "@tanstack/query-core": "workspace:*"
   },
   "peerDependencies": {
-    "solid-js": "^1.6.2"
+    "solid-js": "^1.6.13"
```

Comment:
Bumped for this fix -> https://github.com/solidjs/solid/commit/60f8624db81e5491bdd3f4b69870899976212941